### ISSUE TITLE
Exception thrown on settings page if you don't have any volumes configured

### ISSUE THREAD

**Issue by mtnorthrop** [studioespresso/craft-unsplash#10]
If you install this plugin before configuring any volumes on your site, you get an `Undefined variable: destinationOptions` exception in `craft-splashingimages/src/SplashingImages.php` on line 142:

```
    {
        $volumes = Craft::$app->getVolumes();
        foreach ($volumes->getAllVolumes() as $source) {
            $destinationOptions[] = array('label' => $source->name, 'value' => $source->id);
        }
        return Craft::$app->view->renderTemplate(
            'splashing-images/settings',
            [
                'settings' => $this->getSettings(),
                'volumes' => $destinationOptions,
            ]
        );
    }
```

Of course, this is simply fixed by creating a volume, but I thought I would file this issue in case you would like to create a way of handling this in a way that wouldn't throw an exception.

**Comment by janhenckens** [comment]
Hey! Thanks for catching that! That can certainly be more graceful :)

**Comment by janhenckens** [comment]
Fixed in 2.1.1, which is out now.
